net/api/api.h

```diff
@@ -91,4 +91,6 @@ inline decltype(Singleton<HTTP_IMPL>()(std::declval<T1>(), std::declval<T2>()))
 }  // namespace net
 }  // namespace bricks
 
+using bricks::net::api::HTTP;
```

Comment:
How about `GET`, `POST` and other members of `bricks::net::api`? https://github.com/dkorolev/Bricks/blob/3ac9345832934c8d9b61c7bb91b17a1ee6d59950/net/api/types.h#L70